test/analysis/buildTypeWeights.test.ts

```diff
@@ -2,6 +2,19 @@ import { buildSchema } from 'graphql';
 import { GraphQLSchema } from 'graphql/type/schema';
 import buildTypeWeightsFromSchema from '../../src/analysis/buildTypeWeights';
 
+interface TestFields {
+    [index: string]: number;
+}
+
+interface TestType {
+    weight: number;
+    fields: TestFields;
+}
+
+interface TestTypeWeightObject {
```

Comment:
Might be missing something obvious but why are creating test types here?

Reply:
It's not obvious. I just spent 5 minutes remembering. I made the type alias for the type weights object have `readonly` properties so that no can modify them when using that object in the complexity algorithm. Since some of my tests depended on overwriting the properties I created these type aliases only for testing that do not have the `readonly` restriction.